.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -0,0 +1,144 @@
+name: 🐛 Bug report
+description: Describe a problem
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some general tips:
+
+        - Is this really a problem?
+        - Is this a problem here?
+        - Can this be solved in a different way?
+
+  - type: checkboxes
+    id: initial-checklist
+    attributes:
+      label: Initial checklist
+      options:
+        - label: I read the [support docs](https://mdxjs.com/support)
+          required: true
+        - label: I read the [contributing guide](https://mdxjs.com/contributing)
+          required: true
+        - label: I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
+          required: true
+        - label: I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)
+          required: true
+
+  - type: input
+    id: affected-versions
+    attributes:
+      label: Affected packages and versions
+      description: Please test using the latest version of the relevant packages to make sure your issue has not already been fixed.
+    validations:
+      required: true
+
+  - type: input
+    id: runnable-example
+    attributes:
+      label: Link to runnable example
+      description: |
+        Link to repository or sandbox with runnable example of the issue.
+
+        Some CodeSandbox starters:
+        - [MDX Evaluate with Create React App](https://codesandbox.io/s/github/mdx-js/.github/sandbox-templates/mdx-evaluate-create-react-app)
+        - [MDX Loader with Next.js](https://codesandbox.io/s/github/mdx-js/.github/sandbox-templates/mdx-loader-next)
```

Comment:
Preview URLs:
https://codesandbox.io/s/github/ChristianMurphy/.github-mdx/tree/issue-forms/sandbox-templates/mdx-evaluate-create-react-app
and
https://codesandbox.io/s/github/ChristianMurphy/.github-mdx/tree/issue-forms/sandbox-templates/mdx-loader-next